Assets/Scripts/AI/AIManager.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.AI;
+
+public class AiManager : MonoBehaviour
```

Comment:
Sound like from the name `AiManager` that you're managing multiple AI actors, but from the implementation, it seems like a `AiManager` component has to be attached to each AI actor. Consider renaming the class to something like `AiActor` or even just `Ai`

Reply:
Alright